packages/app/package.json

```diff
@@ -39,7 +39,7 @@
     "@mantine/spotlight": "7.9.2",
     "@microsoft/fetch-event-source": "^2.0.1",
     "@tabler/icons-react": "^3.5.0",
-    "@tanstack/react-query": "^5.56.2",
+    "@tanstack/react-query": "^5.79.2",
```

Comment:
just curious, was this needed? I hesitate to do version upgrades like this unless necessary.